### PR TITLE
helm chart documentation

### PR DESCRIPTION
**Pull request by apoland**
Add documentation under "Gettting Started" for installation on kubernetes via helm chart.

Looking for testing and feedback.

I could see this being its own section instead of under Getting Started.

### CODE REVIEW

**Comment by apoland** [comment]
image

File: docs/getting-started/self-hosted/helm-chart.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+sidebar_label: Deploy to Kubernetes
+title: Deploy Self-Hosted Console to Kubernetes
+description: Deploy Self-Hosted Console to Kubernetes
+---
+
+# Deploy Self-Hosted Console to Kubernetes
+
+## Helm Chart
+
+Aserto provides an official helm chart to deploy authorizer, directory, and console deployments to your kubernetes cluster.  For testing purposes you can start up a local kubernetes cluster using [minikube](https://minikube.sigs.k8s.io/docs/start/), or deploy to a remote cluster.
```
Comment (carabasdaniel) [inline docs/getting-started/self-hosted/helm-chart.mdx]:
```suggestion
Aserto provides an official helm chart to deploy an authorizer, directory, and a web console to your kubernetes cluster.  For testing purposes you can start up a local kubernetes cluster using [minikube](https://minikube.sigs.k8s.io/docs/start/), or deploy to a remote cluster.
```

Comment (carabasdaniel) [reply docs/getting-started/self-hosted/helm-chart.mdx]: Can you please squash the commits ?

Comment (apoland) [reply docs/getting-started/self-hosted/helm-chart.mdx]: Sure thing will squash on merge

**Review by carabasdaniel — APPROVED** [review]

**Pull request merged**